int/api/websites/webcreator.go

```diff
@@ -28,6 +30,25 @@ func CreatePrepareForWebsiteHandler(
 	}
 }
 
+func listArchiveContent(zipReader *zip.Reader) []string {
```

Comment:
You function name is misleading, you should name it following what the code is doing.

My proposal:
```suggestion
func listFileName(zipReader *zip.Reader) []string {
```